front/src/api/today-register-list.api.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from "axios";
+import {baseRoute, routesLinks} from "core";
```

Comment:
Should _baseRoute_ live inside the api folder? or should we keep it in core?

Why do we need routesLinks in this import?